<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>809</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">font-weight: bold</string>
  </property>
  <widget class="QWidget" name="Graph">
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="2">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Preferred</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>138</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="0" column="0" colspan="6">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="range_label">
        <property name="styleSheet">
         <string notr="true">font-weight: bold</string>
        </property>
        <property name="text">
         <string> N= [</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="min_n"/>
      </item>
      <item>
       <widget class="QLabel" name="coma_label">
        <property name="styleSheet">
         <string notr="true">font-weight: bold</string>
        </property>
        <property name="text">
         <string> ,</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="max_n"/>
      </item>
      <item>
       <widget class="QLabel" name="channel_label">
        <property name="styleSheet">
         <string notr="true">font-weight: bold;</string>
        </property>
        <property name="text">
         <string>]    Channel</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="peak_channel"/>
      </item>
      <item>
       <widget class="QLabel" name="counts_label">
        <property name="styleSheet">
         <string notr="true">font-weight: bold</string>
        </property>
        <property name="text">
         <string>Counts</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="peak_counts"/>
      </item>
      <item>
       <widget class="QLabel" name="fwhm_label">
        <property name="styleSheet">
         <string notr="true">font-weight: bold</string>
        </property>
        <property name="text">
         <string> FWHM</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="peak_fwhm"/>
      </item>
     </layout>
    </item>
    <item row="1" column="3" colspan="2" alignment="Qt::AlignHCenter">
     <widget class="QRadioButton" name="noise">
      <property name="text">
       <string>Noise</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0" colspan="4">
     <widget class="PlotWidget" name="graph" native="true"/>
    </item>
    <item row="2" column="0" colspan="3">
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="background_label">
        <property name="styleSheet">
         <string notr="true">font-weight: bold</string>
        </property>
        <property name="text">
         <string>Background =</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="bg1"/>
      </item>
      <item>
       <widget class="QLabel" name="exponent">
        <property name="styleSheet">
         <string notr="true">font-weight: bold</string>
        </property>
        <property name="text">
         <string>e</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="plus_label">
        <property name="styleSheet">
         <string notr="true">font-weight: bold</string>
        </property>
        <property name="text">
         <string>  +</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="bg3"/>
      </item>
      <item>
       <widget class="QLabel" name="variable_label_2">
        <property name="styleSheet">
         <string notr="true">font-weight: bold</string>
        </property>
        <property name="text">
         <string>x  +</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="bg4"/>
      </item>
     </layout>
    </item>
    <item row="3" column="4" colspan="2">
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QListWidget" name="list_widget">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
      <widget class="QPushButton" name="del_item">
       <property name="text">
        <string>Delete Item</string>
       </property>
      </widget>
     </widget>
    </item>
    <item row="1" column="0">
     <spacer name="horizontalSpacer_3">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Preferred</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>148</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="1" column="1">
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="label_12">
        <property name="styleSheet">
         <string notr="true">font-weight: bold</string>
        </property>
        <property name="text">
         <string>    -</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="bg2">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="variable_label_1">
        <property name="styleSheet">
         <string notr="true">font-weight: bold</string>
        </property>
        <property name="text">
         <string>x</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="5">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QPushButton" name="apply">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Apply</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="clear">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Clear</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="2" column="3" colspan="3">
     <spacer name="horizontalSpacer_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Preferred</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>57</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="text">
    <string>Close</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="text">
    <string>Help</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PlotWidget</class>
   <extends>QWidget</extends>
   <header>pyqtgraph</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
